test/SourceKit/InterfaceGen/gen_swift_module_cross_import.swift.response

```diff
@@ -0,0 +1,313 @@
+import SwiftOnoneSupport
+
+public func fromA()
+
+
+// MARK: - B Additions
+
+import B
+
+// Available when B is imported with A
```

Comment:
How does this synthesized comment relate to a user documentation comment, could you add some doc-comments in the test cases?

Reply:
The synthesized comment is printed before it at the moment. Added a test case showing the current behavior.